pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "connect-extension-runner"
-version = "0.0.0"
+version = "26.23"
```

Comment:
Is this intended?

Reply:
No, I will fix it in the next iteration.